### PR TITLE
Migrate settings menu icons of contacts to material design icons

### LINKED ISSUE #2777: Huge design regressions on server master
### Describe the bug

The app looks utterly broken on current server master. This might be caused by recent theming and styling changes.

I also tested stable24 and it is not affected.

### Steps to reproduce

1. Checkout server master.
2. Checkout contacts master.
3. Look at the app in the browser.

### Expected behavior

The app should look the same on server stable24 and master.

### Actual behavior

It looks broken on server master. It does not look broken on server stable24.

Screenshot_20220607_123713

### Contact version

master

### PR DESCRIPTION
**Pull request by GretaD**
ref https://github.com/nextcloud/groupware/issues/38
before
beforesettings
after
aftersettings
fixes #2777

### CODE REVIEW

File: src/components/AppNavigation/Settings/SettingsAddressbook.vue [hunk 1]
```diff
@@ -50,22 +50,25 @@
...
-						icon="icon-rename"
 						@click.stop.prevent="renameAddressbook">
+						<template #icon>
+							<IconRename :size="20" />
+						</template>
 						{{ t('contacts', 'Rename') }}
 					</ActionButton>
 					<ActionInput v-else
 						ref="renameInput"
 						:disabled="renameLoading"
```
Comment (GretaD) [inline src/components/AppNavigation/Settings/SettingsAddressbook.vue:71]: leftovers, will fix asap

File: src/components/AppNavigation/Settings/SettingsAddressbook.vue [hunk 2]
```diff
@@ -115,6 +122,10 @@ export default {
 		ActionInput,
 		ActionLink,
 		Actions,
+		Button,
```
Comment (ChristophWurst) [inline src/components/AppNavigation/Settings/SettingsAddressbook.vue:125]: unused

File: src/components/AppNavigation/Settings/SettingsAddressbook.vue [hunk 3]
```diff
@@ -311,4 +322,11 @@ export default {
 		opacity: .5;
 	}
 }
+.settings-addressbook-list {
+	display: flex;
+	gap: 4px;
+	 li {
```
Comment (ChristophWurst) [inline src/components/AppNavigation/Settings/SettingsAddressbook.vue:328]: indentation

**Comment by szaimen** [comment]
@GretaD Maybe just as a heads on: this is how the settings look currently in non-broken form:
image

**Review by st3iny — APPROVED** [review]
I fixed the fetch-social-avatar checkbox. Otherwise, the PR was good as is.

## How it looks now

I think we came pretty close to the screenshot from before.

ksnip_20220715-143254

**Pull request merged**